packages/api-v4/src/aglb/routes.ts

```diff
@@ -1,61 +1,95 @@
-import Request, { setData, setMethod, setURL } from '../request';
-import { ResourcePage } from 'src/types';
-import { BETA_API_ROOT } from 'src/constants';
+import Request, {
+  setData,
+  setMethod,
+  setParams,
+  setURL,
+  setXFilter,
+} from '../request';
+import { Filter, Params, ResourcePage } from '../types';
+import { BETA_API_ROOT } from '../constants';
 import type { Route, RoutePayload } from './types';
 
 /**
- * getRoutes
+ * getLoadbalancerRoutes
  *
  * Returns a paginated list of Akamai Global Load Balancer routes
  */
-export const getRoutes = () =>
+export const getLoadbalancerRoutes = (
+  id: number,
```

Comment:
```suggestion
  routeId: number,
```
Just for consistency?